Gemfile.lock

```diff
@@ -211,6 +211,9 @@ GEM
       mini_mime (>= 0.1.1)
     marcel (1.0.0)
     method_source (1.0.0)
+    mimemagic (0.3.9)
```

Comment:
why mimemagic?

Reply:
It due to [this issue](https://github.com/rails/rails/issues/41757), I got error when running `bundle install` and upgrading the version fix the problem.